libchisel/src/binaryenopt.rs

```diff
@@ -0,0 +1,107 @@
+/*
+ * Binaryen optimiser.
+ */
+
+use std::collections::HashMap;
+
+use super::{ModuleError, ModulePreset, ModuleTranslator};
+use parity_wasm::elements::*;
+
+// FIXME: change level names
```

Comment:
Perhaps something renaming the enum itself would be clean enough. Something like `BinaryenOptimizationLevel::O1`, etc. would be more clear, I think.